hints/two-integer-sum-ii.md

```diff
@@ -0,0 +1,31 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(n)</code> time and <code>O(1)</code> space, where <code>n</code> is the size of the input array.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 1</summary>
+    <p>
+    A brute force solution would be to check every pair of numbers in the array. This would be an <code>O(n<sup>2</sup>)</code> solution. Can you think of a better way?
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 2</summary>
+    <p>
+    Can you think of an algorithm by taking the advantage of array being sorted?
```

Comment:
Maybe we can be more specific:

If nums[0] + nums[n-1] > target, than we know nums[n-1] can not possibly be included in any pairs. Why? Because nums[n-1] is the largest element in the array. Even by adding it with nums[0], which is the smallest element, we still exceed the target.


---

This hints at a two pointer solution, and also gives some intuition of why it works.